### PR TITLE
[RFR] [BC-Break] Upgrade dependencies

### PR DESCRIPTION
**Pull request by Kmaschta**
:warning: We will remove the support for Node <8

## TODO
- [x] Upgrade serverless and related plugins
- [x] Discontinue Node 6 and support Node >=8
- [x] Babel 7
- [x] Webpack 4
- [x] Other dependencies

### CODE REVIEW

File: cli/package.json [hunk 1]
```diff
@@ -1,53 +1,53 @@
...
+    "description": "Configuration without pain",
+    "keywords": [
+        "configuration",
+        "deployment",
+        "node",
+        "bash",
+        "continuous"
+    ],
+    "engines": {
+        "node": ">=8"
```
Comment (Kmaschta) [inline cli/package.json]: I only changed this line in this file

File: api/package.json [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+        "config": "~2.0.1",
+        "object-hash": "~1.3.0",
+        "slug": "~0.9.1",
+        "uuid": "~3.3.2"
+    },
+    "devDependencies": {
+        "@babel/core": "~7.1.2",
+        "@babel/plugin-transform-async-to-generator": "~7.1.0",
+        "@babel/preset-env": "~7.1.0",
+        "babel-core": "~7.0.0-bridge.0",
```
Comment (Luwangel) [inline api/package.json]: Why do you keep old babel-core?

File: cli/package.json [hunk 3]
```diff
@@ -1,53 +1,52 @@
...
+        "request": "~2.88.0",
+        "traverse": "0.6.6",
+        "uuid": "~3.3.2"
+    },
+    "devDependencies": {
+        "eslint": "~5.6.1",
+        "eslint-config-airbnb": "~17.1.0",
+        "eslint-plugin-import": "~2.14.0",
+        "eslint-plugin-jsx-a11y": "~6.1.1",
+        "eslint-plugin-react": "~7.11.1",
```
Comment (Luwangel) [inline cli/package.json]: Do you use react in cli?

File: api/package.json (hunk 2 above)
Comment (Kmaschta) [reply api/package.json]: It's for jest! It needs this version too (but the special version `bridge`)
https://jestjs.io/docs/en/getting-started#using-babel

**Pull request merged**